lib/views/frontend/spree/checkout/payment/_stripe.html.erb

```diff
@@ -41,7 +41,7 @@
       Spree.stripePaymentMethod.append("<input type='hidden' class='stripeToken' name='payment_source[" + paymentMethodId + "][last_digits]' value='" + response.card.last4 + "'/>");
       Spree.stripePaymentMethod.append("<input type='hidden' class='stripeToken' name='payment_source[" + paymentMethodId + "][month]' value='" + response.card.exp_month + "'/>");
       Spree.stripePaymentMethod.append("<input type='hidden' class='stripeToken' name='payment_source[" + paymentMethodId + "][year]' value='" + response.card.exp_year + "'/>");
-      return Spree.stripePaymentMethod.parents("form").get(0).submit();
+      return Spree.stripePaymentMethod.parents("form").trigger('submit');
```

Comment:
I don't think so, as forms are not nestable so parents("form") should return at most a single element, and Spree.stripePaymentMethod is assigned from an id selector which should return at most a single element (Spree.stripePaymentMethod = $('#payment_method_' + <%= payment_method.id %>)). That being said, it should be harmless to pick out the first element, but I think we should use first() and not get(0), since first() returns a jQuery object we can call trigger() on. I can make this change and resubmit the pull request if you prefer, just let me know, thanks!